test/forms/labels/labeled-fieldset.test.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { LabeledFieldset } from '../../../src'
+
+test('wraps children in fieldset', () => {
+  const Wrapped = () => <input name="test" />
+  const props = { input: { name: 'foo' }, meta: {} }
+  const wrapper = mount(
+    <LabeledFieldset {...props}>
+      <Wrapped />
+    </LabeledFieldset>
+  )
+  expect(wrapper.find('fieldset').exists()).toEqual(true)
+  expect(wrapper.find('fieldset').hasClass('error')).toEqual(false)
```

Comment:
This doesn't align with the test description 🤔 . Can we create a separate test for this?